src/content/operations/cleanup/_index.en.md

```diff
@@ -0,0 +1,105 @@
+---
+title: "Uninstalling Submariner"
+date: 2020-12-23T21:25:11+01:00
+weight: 50
+---
+
+To properly uninstall Submariner from a cluster, follow the steps below:
+
+1. Delete Submariner related namespaces
+
+   On each participating cluster, issue the following command:
```

Comment:
```suggestion
   On each participating cluster:
```

Maybe remove this repetitive line from each section?

Reply:
I kind of like it for the sort of descriptiveness.